### DISCUSSION TITLE
Training Log 2023-01-21

### DISCUSSION THREAD

**Discussion by zh-zheng** [OpenBMB/CPM-Live#315] [Training Logs 训练日志]
# CPM-Live Training Log (January, 21)

Time: January, 21 2023 19:00

Recorder: @zh-zheng

## Loss
- Begin: 2.05
- End: 2.02
	
## Completed Data
- $\approx$ 2243.59GB

## Average Grad Norm
- 1.25

## Progress
- 92.96%

## Comment

We wish everyone a Happy Chinese New Year!

**Comment by zibuyu** [comment]
Thank you for posting the log during the holidays. Happy Chinese new year!
